src/color-scale/color-scale.css

```diff
@@ -1,26 +1,85 @@
-:host {
+[part="wrapper"] {
+	display: grid;
 	gap: .3em;
+	grid-auto-flow: row;
+	grid-template-rows: auto auto;
+	grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
 }
 
 #swatches {
-	display: flex;
-	gap: inherit;
+	display: contents;
+}
+
+[part="add-button"],
+[part="delete-button"] {
```

Comment:
I would suggest using "add button", then people can target them separately too (and you'd be able to simplify this to just `[part~="button"]

Reply:
Excellent idea! Thanks.